### DISCUSSION TITLE
Is there a way to sort consumer groups by messages behind?

### DISCUSSION THREAD

**Discussion by yardenshoham** [provectus/kafka-ui#3467] [Q&A] [answered]
In the UI we can sort by `Group ID`, `Num Of Members` or `State` but not `Num of Topics`, `Messages Behind` nor `Coordinator`.
![image](https://user-images.githubusercontent.com/20454870/224008985-f275e71d-44b6-4234-bb58-e6400ce6d719.png)

**Comment by Haarolean** [comment, marked as answer]
Gonna solve this within #3468
